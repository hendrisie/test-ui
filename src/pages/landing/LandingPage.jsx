import React from 'react';
import { Provider } from 'react-redux';
import configureStore from './store';
import Page from '../../components/Page';
import { ConvertForm, Result } from '../../features/convert'

const store = configureStore();

class LandingPage extends React.Component {
  render() {
    return (
      <Provider store={store}>
        <Page title={"Simple Denominator App"}>
          <ConvertForm />
          <Result />
        </Page>
      </Provider>
    );
  }
}

export default LandingPage;