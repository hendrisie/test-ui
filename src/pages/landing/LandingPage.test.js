import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from './LandingPage';
describe('LandingPage', () => {
  it('whould render correctly', () => {
    const component = shallow(<LandingPage />);
    expect(component).toMatchSnapshot();
  });
});