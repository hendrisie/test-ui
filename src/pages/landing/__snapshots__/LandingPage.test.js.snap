// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage whould render correctly 1`] = `ShallowWrapper {}`;
